var/spack/repos/builtin/packages/r-abadata/package.py

```diff
@@ -7,16 +7,24 @@
 
 
 class RAbadata(RPackage):
-    """Provides the data for the gene expression enrichment analysis
-    conducted in the package 'ABAEnrichment'. The package includes three
-    datasets which are derived from the Allen Brain Atlas: (1) Gene
-    expression data from Human Brain (adults) averaged across donors,
-    (2) Gene expression data from the Developing Human Brain pooled into
-    five age categories and averaged across donors and (3) a developmental
-    effect score based on the Developing Human Brain expression data.
-    All datasets are restricted to protein coding genes."""
+    """Averaged gene expression in human brain regions from Allen Brain Atlas.
 
-    homepage = "https://bioconductor.org/packages/ABAData/"
-    url      = "https://bioconductor.org/packages/release/data/experiment/src/contrib/ABAData_1.14.0.tar.gz"
+       Provides the data for the gene expression enrichment analysis conducted
+       in the package 'ABAEnrichment'. The package includes three datasets
+       which are derived from the Allen Brain Atlas: (1) Gene expression data
+       from Human Brain (adults) averaged across donors, (2) Gene expression
+       data from the Developing Human Brain pooled into five age categories and
+       averaged across donors and (3) a developmental effect score based on the
+       Developing Human Brain expression data. All datasets are restricted to
+       protein coding genes."""
```

Comment:
This is an example of a package that actually benefits from combining the short and long description, although I think the short description is sufficient if you want to simplify things that way.